src/foam/nanos/pii/PIIAware.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license
+ * Copyright 2025 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.pii',
+  name: 'PIIAware',
+
+  methods: [
+    {
+      name: 'piiSummary',
+      type: 'String',
+      code: function() { return this.toSummary(); },
```

Comment:
documentation would help to clarify why this is a sensible default.

Reply:
Removed the default implementation. After a review of many model's toSummary, it's not a good default, it just happened to be on the models I was working with regularly.